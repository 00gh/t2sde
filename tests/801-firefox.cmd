#!/bin/sh

[ "$QEMU" ] || exit 43

exe=opt/mozilla/lib*/firefox/firefox

[ -e $SYSROOT/$exe ] || exit 42

cd $SYSROOT # for wildcard expansion
$QEMU -chroot $SYSROOT $exe --version
