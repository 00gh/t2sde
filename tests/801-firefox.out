Mozilla Firefox 134.0
